src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -958,7 +958,7 @@ export default {
           title: this.filingTypeToName(filingType),
           subtitle,
           filingId: header.filingId,
-          filingAuthor: 'Registry Staff', // TBD
+          filingAuthor: this.filingAuthor(filing),
```

Comment:
The API will now control what is displayed for Transition filings (and Paper filings below), if anything -- this is backward compatible.